packages/server-dev/src/manager/processes/startWatchers.mjs

```diff
@@ -14,50 +14,17 @@
   limitations under the License.
 */
 
-import configWatcher from '../watchers/configWatcher.mjs';
 import envWatcher from '../watchers/envWatcher.mjs';
-
```

Comment:
This breakdown of the process was nice to include / understand. Maybe we can keep / maintain it somewhere?